examples/citi_bike/online_lp/citi_bike_lp.py

```diff
@@ -0,0 +1,187 @@
+import math
+from typing import List, Tuple
+
+import numpy as np
+from pulp import LpProblem, LpVariable, LpMaximize, lpSum, LpStatus, GLPK
+
+from maro.utils import DottableDict
+
+class LP():
```

Comment:
LPSolver?

Reply:
or ILPSolver?

Reply:
Solver indicates the algorithm it used to solve the problem: CBC, CLPK, ... Using ``Solver`` may mess up the concepts here?

Reply:
OK, ILP?

Reply:
It seems PuLP calls it [Integer Programming](https://github.com/coin-or/pulp/blob/bac6d9d2214ba773d638d2de5149940cfd711359/doc/source/main/optimisation_concepts.rst) directly.

I have one more concern for the renaming. We kept naming it as LP, online LP...